cicd/build.sh

```diff
@@ -41,7 +43,13 @@ if test -f /etc/redhat-release && grep -q -i "release 7" /etc/redhat-release; th
     mkdir -p "$DOCKER_CONF"
     docker --config="$DOCKER_CONF" login -u="$QUAY_USER" -p="$QUAY_TOKEN" quay.io
     docker --config="$DOCKER_CONF" login -u="$RH_REGISTRY_USER" -p="$RH_REGISTRY_TOKEN" registry.redhat.io
-    docker --config="$DOCKER_CONF" build -t "${IMAGE}:${IMAGE_TAG}" $APP_ROOT -f $APP_ROOT/$DOCKERFILE
+    if [ "$CACHE_FROM_LATEST_IMAGE" == "true" ]; then
+        docker --config="$DOCKER_CONF" pull "${IMAGE}" && \
+        docker --config="$DOCKER_CONF" build -t "${IMAGE}:${IMAGE_TAG}" $APP_ROOT -f $APP_ROOT/$DOCKERFILE --cache-from "${IMAGE}" || \
```

Comment:
Can you add an 'echo' message here after the `||` to indicate that the build w/ cache failed and you are falling back to use the standard build?

Reply:
I'll merge now and add it